### ISSUE TITLE
deb package

### ISSUE THREAD

**Issue by biokomiker** [bassmanitram/actions-for-nautilus#4]
Dear Martin,

thanks a lot that you started with this project. I use the file manager actions a lot was deeply disappointed that support for nautilus was dropped (it still works in Nemo though).

Anyway I get some warning/error messages when trying to install the deb package via Gdebi, see below:

E: actions-for-nautilus: depends-on-essential-package-without-using-version Suggests: bash
E: actions-for-nautilus: extended-description-is-empty
E: actions-for-nautilus: no-changelog usr/share/doc/actions-for-nautilus/changelog.gz (native package)
E: actions-for-nautilus: no-copyright-file
E: actions-for-nautilus: privacy-breach-uses-embedded-file You may use the libjs-bootstrap package. (https://cdn.jsdelivr.net/npm/bootstrap@latest/dist/css/bootstrap.min.css) [usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.html]
E: actions-for-nautilus: privacy-breach-uses-embedded-file You may use the libjs-jquery package. (https://cdn.jsdelivr.net/npm/jquery@latest/dist/jquery.min.js) [usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.html]
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/ 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/ 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/ 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/README.md 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.desktop 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.html 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.py 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/actions-for-nautilus.schema.json 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/command-line-help.html 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/find-a-port.py 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/sample-config.json 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/start-configurator.sh 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/sub-menu.png 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/actions-for-nautilus-configurator/test.html 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/applications/ 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/applications/actions-for-nautilus-configurator.desktop 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/nautilus-python/ 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/nautilus-python/extensions/ 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/nautilus-python/extensions/actions-for-nautilus.py 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/nautilus-python/extensions/actions-for-nautilus/ 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/nautilus-python/extensions/actions-for-nautilus/actions_for_nautilus.py 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/nautilus-python/extensions/actions-for-nautilus/afn_config.py 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/nautilus-python/extensions/actions-for-nautilus/afn_menu.py 1000/1000
E: actions-for-nautilus: wrong-file-owner-uid-or-gid usr/share/nautilus-python/extensions/actions-for-nautilus/afn_place_holders.py 1000/1000
W: actions-for-nautilus: description-synopsis-starts-with-article
W: actions-for-nautilus: non-standard-dir-perm usr/ 0775 != 0755
W: actions-for-nautilus: non-standard-dir-perm usr/share/ 0775 != 0755
W: actions-for-nautilus: non-standard-dir-perm usr/share/actions-for-nautilus-configurator/ 0775 != 0755
W: actions-for-nautilus: non-standard-dir-perm usr/share/applications/ 0775 != 0755
W: actions-for-nautilus: non-standard-dir-perm usr/share/nautilus-python/ 0775 != 0755
W: actions-for-nautilus: non-standard-dir-perm usr/share/nautilus-python/extensions/ 0775 != 0755
W: actions-for-nautilus: non-standard-dir-perm usr/share/nautilus-python/extensions/actions-for-nautilus/ 0775 != 0755
W: actions-for-nautilus: non-standard-executable-perm usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.desktop 0775 != 0755
W: actions-for-nautilus: non-standard-executable-perm usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.py 0775 != 0755
W: actions-for-nautilus: non-standard-executable-perm usr/share/actions-for-nautilus-configurator/start-configurator.sh 0775 != 0755
W: actions-for-nautilus: non-standard-file-perm usr/share/actions-for-nautilus-configurator/README.md 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.html 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/actions-for-nautilus-configurator/actions-for-nautilus.schema.json 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/actions-for-nautilus-configurator/command-line-help.html 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/actions-for-nautilus-configurator/find-a-port.py 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/actions-for-nautilus-configurator/sample-config.json 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/actions-for-nautilus-configurator/sub-menu.png 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/actions-for-nautilus-configurator/test.html 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/applications/actions-for-nautilus-configurator.desktop 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/nautilus-python/extensions/actions-for-nautilus.py 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/nautilus-python/extensions/actions-for-nautilus/actions_for_nautilus.py 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/nautilus-python/extensions/actions-for-nautilus/afn_config.py 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/nautilus-python/extensions/actions-for-nautilus/afn_menu.py 0664 != 0644
W: actions-for-nautilus: non-standard-file-perm usr/share/nautilus-python/extensions/actions-for-nautilus/afn_place_holders.py 0664 != 0644
W: actions-for-nautilus: privacy-breach-generic [<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/spectre.css@latest/dist/spectre-icons.min.css">] (https://cdn.jsdelivr.net/npm/spectre.css@latest/dist/spectre-icons.min.css) [usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.html]
W: actions-for-nautilus: privacy-breach-generic [<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">] (https://use.fontawesome.com/releases/v5.6.3/css/all.css) [usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.html]
W: actions-for-nautilus: privacy-breach-generic [<link rel="stylesheet"\n??href="https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/css/jsoneditor.min.css">] (https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/css/jsoneditor.min.css) [usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.html]
W: actions-for-nautilus: privacy-breach-generic [<script src="https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/nonmin/jsoneditor.js">] (https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/nonmin/jsoneditor.js) [usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.html]
W: actions-for-nautilus: script-not-executable [usr/share/applications/actions-for-nautilus-configurator.desktop]
W: actions-for-nautilus: synopsis-too-long
W: actions-for-nautilus: unknown-section GNOME Desktop Environment
W: actions-for-nautilus: unusual-interpreter /usr/bin/python [usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.py]
W: actions-for-nautilus: unusual-interpreter /usr/bin/python [usr/share/actions-for-nautilus-configurator/find-a-port.py]
W: actions-for-nautilus: unusual-interpreter xdg-open [usr/share/actions-for-nautilus-configurator/actions-for-nautilus-configurator.desktop]
W: actions-for-nautilus: unusual-interpreter xdg-open [usr/share/applications/actions-for-nautilus-configurator.desktop]

Lintian finished with exit status 0`

Did not have that so far.

Anyway a big THANK YOU so far.
`

**Comment by bassmanitram** [comment]
Thanks for the info - i'll look at the errors and try to fix them (I'm on
Pop-OS, so it's Pop-on-Ubuntu-on-Deb - Im also a bit of a poweruser so may
be assuming too many things!

Can you tell me what distribution and version you are using?

**Comment by biokomiker** [comment]
I am now on Ubuntu 22.04 / Gnome 42.

**Comment by bassmanitram** [comment]
OK, that's mine too (kinda - Pop-os->Ubuntu 22.04)

Anyway, much progress is being made down to three messages from Lintian - copyright file and changelog missing errors, then unknown section warning for `GNOME Desktop Environment`.

Should have this purified by the end of the day - thanks for the report

**Comment by bassmanitram** [comment]
OK ,PR: https://github.com/bassmanitram/actions-for-nautilus/pull/7 is ready - @biokomiker can I ask you to test the deb package install: https://github.com/bassmanitram/actions-for-nautilus/raw/issue/0004/dist/actions-for-nautilus_1.0.2~rc1_all.deb

**Comment by biokomiker** [comment]
Looks very find now.

Gdebi output:

```
W: actions-for-nautilus: copyright-without-copyright-notice

Lintian finished with exit status 0
```

Uninstalled user install

Installed deb via Gdebi

all working so far.

Thanks

**Comment by bassmanitram** [comment]
[PR](https://github.com/bassmanitram/actions-for-nautilus/pull/7) 
Fixed in [Release 1.0.2](https://github.com/bassmanitram/actions-for-nautilus/releases/tag/v1.0.2)
